src/elements/netspeak-search.tsx

```diff
@@ -484,6 +519,7 @@ export class NetspeakSearch extends React.PureComponent<Props, State> {
 							phrases={this.state.phrases}
 							stats={this.state.phrasesStats}
 							onChange={this._onPhraseChange}
+							showExperimental={this.props.showExperimental}
```

Comment:
This can be removed.

`showExperimental` isn't actually used anywhere in `NetspeakResultList`, so you can remove this and all mentions of `NetspeakResultList` in `netspeak-result-list.tsx`.